OracleUnifiedDirectorySM/dockerfiles/12.2.1.4.0/patches/patches.download

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at 
+# https://oss.oracle.com/licenses/upl.
+#
+# Oracle Unified Directory
+# Download patches from : https://support.oracle.com
+#
+# To build the OUD image with patches, you must download and drop the OUD 12.2.1.4.0 patch zip files (for e.g. p30851280_122140_Generic.zip) into the patches directory.
```

Comment:
Is this file necessary? It doesn't contain any actual file names or md5 checksums, so there is no value provided.

Reply:
Until README is updated, based on this "To build the OUD image with patches, you must download and drop the OUD 12.2.1.4.0 patch zip files (for e.g. p30851280_122140_Generic.zip) into the patches directory." in patches.download file, we are trying to convey that patch zip files can be kept in the folder.

Reply:
If there are no patches, there is no need for this folder or file. Remove them for now and when you _do_ have patches, add them at that point. If there is no use for something, it shouldn't be included.